src/lib/crypto/kimchi_bindings/stubs/src/arkworks/bigint_256.rs

```diff
@@ -247,4 +282,42 @@ mod tests {
         assert!(x2 == x);
         println!("biguint.to_string: {}", x2.to_string());
     }
+
+    #[test]
+    fn serialization() {
+        let x = 10000.to_biguint().unwrap();
+        let x = BigInteger256::try_from(x).unwrap();
+
+        debug_assert_eq!(std::mem::size_of::<BigInteger256>(), 32);
+        let mut bytes = [0u8; std::mem::size_of::<BigInteger256>()];
+        x.serialize_compressed(&mut bytes[..]).unwrap();
+
+        println!("bytes: {:?}", bytes);
+        assert_eq!(
+            bytes,
+            [
+                16u8, 39, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                0, 0, 0, 0, 0, 0
+            ]
+        );
+
+        let y = BigInteger256::deserialize_compressed(&bytes[..]).unwrap();
+
+        assert_eq!(x, y);
+    }
```

Comment:
I had to spend so much time figuring out what had changed in the serialization that I added some tests to make sure breaking changes will be noticed in the future